pegasus/sites.v3/code.org/views/mobile_footer_responsive.haml

```diff
@@ -1,5 +1,5 @@
 %link{:href=>'/css/mobile-footer.css', :rel=>'stylesheet'}
-%script{src: '/js/mobile-footer.js'}
+%script{:defer => "defer", src: '/js/mobile-footer.js'}
```

Comment:
This doesn't seem to have any dependencies (and nothing depends on it), so this should be okay (and could even be `async` instead of `defer`,  though this is not necessary).